basic-fetch/index.html

```diff
@@ -19,26 +16,25 @@ <h1>Fetch basic example</h1>
     <img src="" class="my-image">
   </body>
   <script>
-    var myImage = document.querySelector('.my-image');
+    const myImage = document.querySelector('.my-image');
 
     fetch('flowers.jpg')
-    .then(function(response) {
+    .then((response) => {
```

Comment:
Since we're at it, can we run Prettier on these examples? The indentation is making me feel itchy.

Reply:
Good idea. Done.